src/gui/styles/dark/DarkStyle.cpp

```diff
@@ -23,8 +23,9 @@
 #include <QMenuBar>
 #include <QToolBar>
 
-void DarkStyle::polish(QPalette& palette)
+QPalette DarkStyle::standardPalette() const
 {
+    auto palette = BaseStyle::standardPalette();
     palette.setColor(QPalette::Active, QPalette::Window, QStringLiteral("#3B3B3D"));
     palette.setColor(QPalette::Inactive, QPalette::Window, QStringLiteral("#404042"));
     palette.setColor(QPalette::Disabled, QPalette::Window, QStringLiteral("#424242"));
```

Comment:
Since you're at it, you should replace all these QStringLiterals with `QRgb` hex numbers like `QRgb(0xABCDEF)`.